spec/requests/spaces_controller_request_spec.rb

```diff
@@ -124,6 +124,25 @@
         expect(flash[:notice]).to include("successfully updated")
       end
     end
+
+    context "when a Operator" do
+      let(:space) { create(:space, :with_members, :with_rooms) }
+      let(:operator) { create(:person, operator: true) }
+
+      it "enables Enforce SSL on a Space" do
+        new_entrance = space.rooms.sample
+        put polymorphic_path(space), params: {space: {entrance_id: new_entrance.id, enforce_ssl: 1}}
+
+        expect(space.enforce_ssl).to be true
+      end
+
+      it "disables Enforce SSL on a Space" do
+        new_entrance = space.rooms.sample
+        put polymorphic_path(space), params: {space: {entrance_id: new_entrance.id, enforce_ssl: 0}}
+
+        expect(space.enforce_ssl).to be false
```

Comment:
Tests still fail. IDK ahy.

I see the `Space` record updating `enforce_ssl` to the correct value  in Puma logs when submitting the form on local.

Reply:
Both the `Controller` `space`  instance and the `spec` `space` instance have the same `id` (since they're referencing the same piece of data where it's stored in the database) but will have a different location in memory; so changes to the `space` instance in the `Controller` will only be reflected in `space` instance in the `spec` once the `space` instance in the `spec` re-queries the database to get the updated value.

TL/DR: You may want to call `space.reload` after executing the `put` because the computer caches the data in the `Space` record independently between the `Controller` and `spec` until it is re-queried..